docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -40,7 +40,7 @@ For the supervised training we use the following algorithms components:
 Dataset Format
 **************
 
-For the dataset handling inside OTX we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
+For the dataset handling inside OpenVINO™ Training Extensions we use `Dataset Management Framework (Datumaro) <https://github.com/openvinotoolkit/datumaro>`_.
```

Comment:
Can you add ,? For the dataset handling inside OpenVINO™ Training Extensions, we use